inc/itilsolution.class.php

```diff
@@ -325,7 +325,7 @@ function post_addItem() {
       }
 
       $item = $this->item;
-      $status = $item::CLOSED;
+      $status = $item::SOLVED;
```

Comment:
Need to change line https://github.com/glpi-project/glpi/pull/4836/files#diff-c1fd7f23e92e1731809fc183d9edc997R338 to `$status = $item::CLOSED;`.

Reply:
Or not, autoclose stuff should maybe not be applied to Problem and Change... 

Anyways, tests will fail, I've opened this PR since I cannot cherry-pick, and I'll forget otherwise :D